### ISSUE TITLE
lack of documentation

### ISSUE THREAD

**Issue by dailycommit** [max-mapper/google-cloud-storage#1]
The document is too inadequate to use this module.
is it just cli ? How do I use this with abstract-blob-store ?
Is there any example like as "s3-blob-store" ?

**Comment by loganmay** [comment]
I also agree it would be helpful to have more documentation. An ideal example for my use case would be demonstrating how to read in data from cloud storage as a stream, pipe that stream into a function where you modify the data from the stream, and then write it back to cloud storage as a stream.

I.e. using this library to accomplish [the streaming described here in the Cloud Platform Docs](https://cloud.google.com/storage/docs/streaming)
